src/main/java/sirius/biz/protocol/Protocols.java

```diff
@@ -128,7 +128,7 @@ public void handle(Incident incident) throws Exception {
                 si.getMdc().put(t.getFirst(), t.getSecond());
             }
             si.setUser(UserContext.getCurrentUser().getProtocolUsername());
-            si.setMessage(incident.getException().getMessage());
+            si.setMessage(Strings.limit(incident.getException().getMessage(), maxMessageLength, true));
```

Comment:
Please give `si` a better name (like `storedIncident`) – as we are touching the code already.